registry/handlers/manifests.go

```diff
@@ -443,7 +443,8 @@ func (imh *manifestHandler) DeleteManifest(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	err = manifests.Delete(imh, imh.Digest)
+	tagService := imh.Repository.Tags(imh)
+	referencedTags, err := tagService.Lookup(imh, distribution.Descriptor{Digest: imh.Digest})
```

Comment:
The below `switch err` is not required for tag lookup api, right?

Reply:
yes, but if i do without ``swith err`` , than i dont pass the test case.